File: web/src/app/core/DTOs/paquete.dto.ts
import { EstadoPaquete } from "../enums/estado-paquete.enum";
import { Prioridad } from "../enums/prioridad.enum";
import { RepartidorDto } from "./repartidor.dto";

export interface PackageDto {
    id: string;
    descripcion: string;
    peso: number;
    codigo: string;
    estado: EstadoPaquete;
    prioridad: Prioridad;
    repartidor: RepartidorDto | null;
}

export interface CreatePackageDto {
    descripcion: string;
    peso: number;
    codigo: string;
    prioridad: Prioridad;
}

File: backend/FlashLogistic.Application/DTOs/Paquetes/PaqueteDTO.cs
using FlashLogistic.Application.DTOs.Repartidor;

namespace FlashLogistic.Application.DTOs.Paquetes;

public class PaqueteDTO
{
    public Guid Id { get; set; }
    public string Descripcion { get; set; } = default!;
    public double Peso { get; set; }
    public string Estado { get; set; }
    public string Prioridad { get; set; }
    public RepartidorDTO? Repartidor { get; set; }
}
